xlsx2csv.py

```diff
@@ -1115,8 +1109,7 @@ def main():
         (options, args) = parser.parse_args()
         if len(args) < 1:
             parser.print_usage()
-            sys.stderr.write("error: too few arguments" + os.linesep)
-            sys.exit(1)
+            raise XlsxException("error: too few arguments" + os.linesep)
```

Comment:
In main, don't trow exceptions but print the error message to stderr

Reply:
Thanks! Curious of the motivation here if you don't mind sharing 😊

Reply:
main is used when you run xlsx2csv as command line application. So a nicely written message in a proper formatted way looks the best in the terminal. When xls2csv is imported as a library, main is not called (only convert, which raises exceptions which can be handled).

Reply:
makes sense 👍 thanks for explaining